lwt_luv.opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+
+synopsis: "Libuv engine for Lwt"
+
+version: "1.0.0"
+license: "MIT"
+homepage: "https://github.com/ocsigen/lwt"
+doc: "https://ocsigen.org/lwt"
+bug-reports: "https://github.com/ocsigen/lwt/issues"
+
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Ulrik Strid <ulrik.strid@outlook.com>"
+]
+dev-repo: "git+https://github.com/ocsigen/lwt.git"
+
+depends: [
+  "lwt"
```

Comment:
```suggestion
  "lwt" {=version}
```

Reply:
We (me and @aantron) decided to make the version of the engine package `1.0.0`, and since it should work with any `Lwt` verison and it probably won't ever get to opam in this form I don't think this change make sense.